packages/twenty-server/src/engine/metadata-modules/serverless-function/serverless-function.service.ts

```diff
@@ -133,8 +144,8 @@ export class ServerlessFunctionService extends TypeOrmQueryService<ServerlessFun
       );
 
       if (
-        serverlessFunctionCreateHash(latestCode || '') ===
-        serverlessFunctionCreateHash(draftCode || '')
+        serverlessFunctionCreateHash(JSON.stringify(latestCode)) ===
+        serverlessFunctionCreateHash(JSON.stringify(draftCode))
```

Comment:
The properties order might not be the same in the `latestCode` and `draftCode` objects even if the values are equal. The specification of `JSON.stringify` doesn't guarantee the order of the properties of an object.

If the has is not stored in the database and is only useful at this exact place of the code, we might prefer to use a `deep-equal` function coming from a library.